nixos/modules/services/cluster/kubernetes.nix

```diff
@@ -599,7 +752,7 @@ in {
         description = "Kubernetes Kubelet Service";
         wantedBy = [ "kubernetes.target" ];
         after = [ "network.target" "docker.service" "kube-apiserver.service" ];
-        path = with pkgs; [ gitMinimal openssh docker utillinux iproute ethtool thin-provisioning-tools iptables ];
+        path = with pkgs; [ gitMinimal openssh docker utillinux iproute ethtool thin-provisioning-tools iptables ] ++ cfg.path;
```

Comment:
yes, that and also ceph binary for distributed storage solution (rook.io), but that is for the next PR